_includes/section_header.html

```diff
@@ -4,4 +4,10 @@
       <a href="{{ site.baseurl }}/discovery">Discovery</a>
     </h1>
   </header>
+  {% elsif page.section == 'alpha' %}
+  <header class="section">
+    <h1>
+      <a href="{{ site.baseurl }}/alpha">Alpha</a>
```

Comment:
There should be a 'draft' label here for the time being - we probably need to add a little bit of styling to it too.